test/parallel/test-repl-top-level-await.js

```diff
@@ -113,7 +113,8 @@ async function ordinaryTests() {
     ['for (let i = 0; i < 2; ++i) await i'],
     ['for (let i = 0; i < 2; ++i) { await i }'],
     ['await 0', '0'],
-    ['await 0; function foo() {}'],
+    ['await 0; function foo() {}',
+     "Uncaught SyntaxError: Identifier 'foo' has already been declared"],
```

Comment:
This seems wrong to me - I think the reason for the issue is that functions are supposed to hoist as "var" not "let".

Reply:
Good catch!, it didn't come to my mind to hoist using "var".